### PR TITLE
Updated prime_numbers.py testcases.

### PR DESCRIPTION
**Pull request by Muhammadummerr**
### Describe your change:
I reduced the test cases, so it will run faster than before.

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [x] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

File: maths/prime_numbers.py [hunk 1]
```diff
@@ -97,9 +97,9 @@ def benchmark():
     from timeit import timeit
 
     setup = "from __main__ import slow_primes, primes, fast_primes"
-    print(timeit("slow_primes(1_000_000_000_000)", setup=setup, number=1_000_000))
-    print(timeit("primes(1_000_000_000_000)", setup=setup, number=1_000_000))
-    print(timeit("fast_primes(1_000_000_000_000)", setup=setup, number=1_000_000))
+    print(timeit("slow_primes(1_000)", setup=setup, number=1_000))
+    print(timeit("primes(1_000)", setup=setup, number=1_000))
+    print(timeit("fast_primes(1_000)", setup=setup, number=1_000))
```
Comment (cclauss) [inline maths/prime_numbers.py:102]: Please revert these three lines.  pytest does not run the code in `__main__()` so it does not run the benchmarks.

**Review by cclauss — APPROVED** [review]

**Comment by cclauss** [comment]
Well done!  Thanks.

**Comment by Muhammadummerr** [comment]
Thank you. It means a lot to me.

**Comment by tianyizheng02** [comment]
Contributes to #9718 (adding for linking purposes)

**Pull request merged**